api/server/server.go

```diff
@@ -1631,7 +1639,27 @@ func ServeApi(job *engine.Job) error {
 	return nil
 }
 
-func toBool(s string) bool {
-	s = strings.ToLower(strings.TrimSpace(s))
-	return !(s == "" || s == "0" || s == "no" || s == "false" || s == "none")
+func containerCommit(eng *engine.Engine, name string, c *daemon.ContainerCommitConfig) (string, error) {
```

Comment:
it used to be part of the daemon actually. But it created a cycled dependency because daemon calls builder that calls daemon. I'll see if we can move it to builder.